phoenix-core/src/main/java/org/apache/phoenix/schema/TableProperty.java

```diff
@@ -182,7 +182,7 @@ public Object getPTableValue(PTable table) {
 
     },
 
-    COLUMN_ENCODED_BYTES(PhoenixDatabaseMetaData.ENCODING_SCHEME, COLUMN_FAMILY_NOT_ALLOWED_TABLE_PROPERTY, false, false, false) {
+    COLUMN_ENCODED_BYTES(PhoenixDatabaseMetaData.ENCODING_SCHEME, COLUMN_FAMILY_NOT_ALLOWED_TABLE_PROPERTY, true, false, false) {
```

Comment:
Nice one, curious any test was failing to validate table props without this change earlier?

Reply:
This property was used for immutable tables only I think. Now we changed the behavior so that mutable indexes can benefit from this. It is not a property reserved for immutable tables. And we didn't have any test to use this property on mutable tables/indexes I think